### ISSUE TITLE
Default description of DNSManagedZone is "Managed by Terraform"

### ISSUE THREAD

**Issue by errordeveloper** [GoogleCloudPlatform/k8s-config-connector#254]
**Describe the bug**

I haven't set the description, but it's being set somehow, and it's not "Managed by Config Connector", but "Managed by Terraform", and it's confusing, since we aren't using Terraform for this (albeit we use it elsewhere).

**ConfigConnector Version**
Run the following command to get the current ConfigConnector version
```bash 
kubectl get ns cnrm-system -o jsonpath='{.metadata.annotations.cnrm\.cloud\.google\.com/version}' 
```

**To Reproduce**
*Steps to reproduce the behavior:*

*YAML snippets:*

```yaml
apiVersion: dns.cnrm.cloud.google.com/v1beta1
kind: DNSManagedZone
metadata:
  name: foo-example-com
  namespace: dns
spec:
  dnsName: "foo.example.com."
```
And after it's create, inside the API it shows up like this:
```
$ kubectl get -n dns DNSManagedZone  -o yaml foo-example-com
apiVersion: dns.cnrm.cloud.google.com/v1beta1
kind: DNSManagedZone
metadata:
  name: foo-example-com
  namespace: dns
spec:
  description: Managed by Terraform
  dnsName: foo.example.com.
  visibility: public
status: {}
```

**Comment by caieo** [comment]
Haha, oh boy. Thanks for bringing this up, @errordeveloper! Because we use some aspects of Terraform, this default field seems to have bubbled up. This is a bug we'll fix. Just to get your input, would it be preferred to have the description be "Managed by Config Connector" or to have the description be empty if the user does not set it?

**Comment by errordeveloper** [comment]
I guess, I would prefer it to be empty =)

**Comment by kibbles-n-bytes** [comment]
This is fixed as of 1.20.1. We updated the description to say instead `Managed by Config Connector`, as the underlying field is required by the API and making it required in Config Connector would be a breaking change.
